package exception;

public class tryCatch {
    public static void main(String[] args) {
    int j = 0;
    int i = 0;
    try{
         j = 18/i;
    }
    catch(Exception e){
        System.out.println(e);
    }
    }
}
